### PR TITLE
Format files with prettier

### PR DESCRIPTION
**Pull request by jasongrout**
This follows up on #2733 and actually reformats files.

This will probably cause lots of conflicts. However, if we merge #2733 first, a PR can be updated to that PR's merge commit, then run prettier on the conflicting files and commit the changes, and that should take care of the conflicts.

### CODE REVIEW

**Comment by jasongrout** [comment]
The last commit here may need to be redone if more things are merged into master before this gets in. It's easy, though:

```
git reset --hard 13b6b01 
yarn run lint
git commit -a
```

File: packages/schema/jupyterwidgetmodels.latest.md [hunk 1]
```diff
@@ -6,1153 +6,1151 @@ widget is using. A reference to a widget is serialized to JSON as a string of
 the form `"IPY_MODEL_<MODEL_ID>"`, where `<MODEL_ID>` is the model ID of a
 previously created widget of the specified type.
 
-This model specification is for ipywidgets 7.4.*, @jupyter-widgets/base 1.1.*,
-and @jupyter-widgets/controls 1.4.*.
+This model specification is for ipywidgets 7.4._, @jupyter-widgets/base 1.1._,
+and @jupyter-widgets/controls 1.4.\*.
```
Comment (SylvainCorlay) [inline packages/schema/jupyterwidgetmodels.latest.md]: Maybe we should drop markdown.

Comment (jasongrout) [reply packages/schema/jupyterwidgetmodels.latest.md]: Done, over in #2733 and this is rebased on that.

**Comment by vidartf** [comment]
> +14,883 −13,537

Oh wow

**Comment by vidartf** [comment]
Note that lints fail on Windows:

```
$ prettier --ignore-path .gitignore --write '**/*{.ts,.tsx,.js,.jsx,.css,.json}'
[error] No matching files. Patterns tried: '**/*{.ts,.tsx,.js,.jsx,.css,.json}' !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**
error Command failed with exit code 2.
```

**Comment by jasongrout** [comment]
> Note that lints fail on Windows:

Ugh. Any idea why?

**Comment by vidartf** [comment]
Wrong quotes for globs, I'll do a PR

**Pull request merged**